mne/utils.py

```diff
@@ -2089,7 +2089,9 @@ def _check_fname(fname, overwrite=False, must_exist=False):
     if must_exist and not op.isfile(fname):
         raise IOError('File "%s" does not exist' % fname)
     if op.isfile(fname):
-        if not overwrite:
+        if overwrite == 'read':
```

Comment:
Ahh I see what you mean now, the `logger.info`

This can be simplified by, instead of adding this, doing `elif overwrite != 'read'` below.

Also, there are 8 other places revealed by git grep ".*check_fname(.*overwrite=True"` that could also benefit from this change